app/design/frontend/base/default/template/ebizmarts/mailchimp/popup/emailcatcher.phtml

```diff
@@ -219,7 +219,9 @@
         }
         <?php endif; ?>
         function validateEmail(email) {
-            var re = /^(([^<>()[\]\\.,;:\s@\"]+(\.[^<>()[\]\\.,;:\s@\"]+)*)|(\".+\"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+            var user = '([^<>()[\\]\\\\.,;:\\s@\\"]+(\\.[^<>()[\\]\\\\.,;:\\s@\\"]+)*)|(\\".+\\")';
+            var domain = '(\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,})';
+            var re = new RegExp('ˆ' + user + '@' + domain + '$');
```

Comment:
Here was solved:
`Line exceeds 120 characters in Reg-Exp (emailcatcher.phtml).`
 ref. #779

Reply:
Is this fully tested?
Can the "new RegExp" instantiation be avoided?

Reply:
Confirmed. The current regular expression used for validating email is the same used before, but divided in two parts: user and domain.